drivers/sensor/lsm6dsl/lsm6dsl.h

```diff
@@ -689,12 +688,20 @@
 #endif
 
 struct lsm6dsl_config {
-	char *i2c_master_dev_name;
-	u16_t i2c_slave_addr;
+	char *comm_master_dev_name;
+};
+
+struct lsm6dsl_data;
+
+struct lsm6dsl_transfer_function {
+	int (*read_data)(struct lsm6dsl_data *data, uint8_t reg_addr, uint8_t *value, uint8_t len);
+	int (*read_reg)(struct lsm6dsl_data *data, uint8_t reg_addr, uint8_t *value);
+	int (*update_reg)(struct lsm6dsl_data *data, uint8_t reg_addr, uint8_t value, uint8_t mask);
```

Comment:
beware of the 80 chars limit 

apply the relevant change everywhere where relevant.

Reply:
ack